internal/command/checks/checks.go

```diff
@@ -0,0 +1,49 @@
+package checks
```

Comment:
It would probably be better to have each subcommand in it's own file and then add them like so: https://github.com/superfly/flyctl/blob/832c567731a1ea687a83af76191ff20e767eb23e/internal/command/machine/machine.go#L26-L38. It's no big E though.